src/components/ExtractingFormOcrControl.vue

```diff
@@ -56,10 +60,14 @@ export default {
         this.textLanguages = textLanguages
         this.ocrLanguages = ocrLanguages
       } catch (e) {
-        this.$root.$bvToast.toast(this.$t('extractingLanguageFormControl.failedToRetrieveLanguages'), {
-          noCloseButton: true,
-          variant: 'danger'
-        })
+        if (e.response.status === 503) {
+          this.$set(this, 'hasTesseract', false)
```

Comment:
for consistency I would have used:

```
this.hasTesseract = false
```

Or even better:

```
this.hasTesseract = e.response.status !== 503

if (this.hasTesseract) {
  this.$root.$bvToast.toast(this.$t('extractingLanguageFormControl.failedToRetrieveLanguages'), {
    noCloseButton: true,
    variant: 'danger'
  })
}
```

Reply:
It works better yes